modules/tinymce/src/core/main/ts/api/AddOnManager.ts

```diff
@@ -12,71 +12,73 @@ import Editor from './Editor';
 import I18n from './util/I18n';
 
 /**
- * This class handles the loading of themes/plugins or other add-ons and their language packs.
+ * TinyMCE theme manager class. Subclass of AddOnManager. Allows for a custom theme to be used with TinyMCE.
  *
- * @class tinymce.AddOnManager
- */
-
-/**
- * TinyMCE theme class.
+ * @summary <strong>Warning</strong>: Much of TinyMCE's functionality is provided by the default Silver theme.
+ * Creating a custom theme may require implementing this functionality.
+ * To change TinyMCE's appearance, Tiny recommends changing the Skin instead.
  *
- * @class tinymce.Theme
- */
-
-/**
- * This method is responsible for rendering/generating the overall user interface with toolbars, buttons, iframe containers etc.
+ * @class tinymce.ThemeManager
```

Comment:
```suggestion
 * @class tinymce.Theme
```
`ThemeManager` isn't really a class, it's just a `AddOnManager` instance, so I think we should just focus on that's needed to define a theme here (more suggestions incoming).